flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1117,6 +1117,9 @@
     "org.kde.kdevelop": {
 	"finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "org.mozilla.Thunderbird": {
+        "finish-args-contains-both-x11-and-wayland": "fallbacks to wayland on non-x11 system"
```

Comment:
Do you mean it should be re-worded somehow? Could you propose something?

Reply:
I don't know what your rationale is. It's your rationale.

Reply:
My rationale is what I put in comment: it uses wayland when there is no x11. I made it bit longer in commit description. Should I match exemption comment with commit description?

Reply:
@hfiguiere t-bird (or every mozilla app) is unique because it uses wayland as secondary choice (not primary as most apps do) only when no x11 socket (xwayland) is available. fallback-x11 artificially creates such environment but the developers intent is to use x11 even on wayland environment unless there is no choice.

This means removing wayland is no-go because t breaks x11 free system when those are supported and fallback-x11 is no go because it forces wayland on system that support both which is considered inferior. Something like fallback-wayland would be needed but it doesn't really make sense for flatpak to have special case for that.